code/game/machinery/computer/cloning.dm

```diff
@@ -210,14 +210,17 @@
 	var/mob/living/mob_occupant = get_mob_or_brainmob(occupant)
 	var/datum/dna/dna
 	var/datum/bank_account/has_bank_account
+
+	// Do not use unless you know what they are.
+	var/mob/living/carbon/C = mob_occupant
+	var/mob/living/brain/B = mob_occupant

```

Comment:
might as well define those right after the respective if checks

Reply:
they're used in multiple different if checks though